tests/ballerina-test-integration/src/test/resources/observability/metrics/metrics-test.bal

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/sql;
+import ballerina/jdbc;
+
+endpoint http:Listener testEp {
+    port:9090
+};
+
+endpoint jdbc:Client testDB {
+    url: "jdbc:hsqldb:hsql://localhost:9001/TEST_DB",
+    username: "SA",
+    poolOptions: {maximumPoolSize:10}
+};
+
+@http:ServiceConfig {
+    basePath:"/"
```

Comment:
It's always better to have a service context, rather have it as a root context. Can you please change this?